src/DynamoCoreWpf/ViewModels/Search/BrowserItemViewModel.cs

```diff
@@ -152,6 +153,22 @@ public string Assembly
             }
         }
 
+        public ElementTypeEnum ElementType
+        {
+            get
+            {
+                if (entries.All(entry => entry.ElementType == ElementTypeEnum.Package) &&
+                    subCategories.All(subCategory => subCategory.ElementType == ElementTypeEnum.Package))
+                    return ElementTypeEnum.Package;
+                else
+                    if (entries.All(entry => entry.ElementType == ElementTypeEnum.CustomDll) &&
+                        subCategories.All(subCategory => subCategory.ElementType == ElementTypeEnum.CustomDll))
+                        return ElementTypeEnum.CustomDll;
+                    else
+                        return ElementTypeEnum.RegularCategory;
+            }
+        }
```

Comment:
I think next structure:
Core -> ElementTypes.BuiltIn
Core, inside of which added some custom node -> ElementTypes.None
Category with just user's custom nodes -> ElementTypes.CustomNode
Category with just user's imported assembly -> ElementTypes.ZeroTouch
Rhynamo -> ElementTypes.Packaged

I think we can use None for mixed categories...